### DISCUSSION TITLE
People Albums

### DISCUSSION THREAD

**Discussion by DevGoran** [immich-app/immich#5444] [Feature Request]
I believe it would be quite nice to have the ability to create albums by choosing people or add people to albums. When then new photos get added to immich, and they belong to a person which are in one of those albums, those photos should get added automatically to those albums.

This is particularly nice if there are groups of people one would like to have in albums, such as an album for friends, family, work, etc.  Otherwise, albums such as these need to be maintained heavily manually.

**Comment by bo0tzz** [comment]
Duplicate of  #1673
